### DISCUSSION TITLE
[Feature] filter by active contributors 🔎

### DISCUSSION THREAD

**Discussion by gianpaj** [community/community#59524] [Repositories]
### Select Topic Area

Product Feedback

### Feature Area

Issues

### Body

In popular and active GitHub repositories, I would like to follow what the contributors are working on and discussing:

https://github.com/ggerganov/llama.cpp

While I can filter by a single author, e.g. the repo owner `ggerganov`: https://github.com/ggerganov/llama.cpp/issues/created_by/ggerganov

I would like to filter by all of those active contributors.

The definition of `active contributors` can be discussed.
Perhaps we could consider the top 10 contributors based on their commits in the last year? https://github.com/ggerganov/llama.cpp/graphs/contributors

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @gianpaj ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
